src/App.tsx

```diff
@@ -74,7 +40,7 @@ const Grid = ({
 }) => {
 	console.log('incoming data', items)
 	return (
-		<div className=" flex flex-wrap max-w-[1500px]">
+		<div className="flex flex-wrap w-[150rem]">
```

Comment:
Setting an explicit width or else when the browser window is a different size the rows/columns of boxes change, messing up the pixel art. We can change this width value to however wide you want want it to be depending on the number of boxes you want. This is 100 boxes wide because each box is 1.5rem